frontend/src/services/Storage.js

```diff
@@ -0,0 +1,52 @@
+const tokenKeyName = 'auth.access_token';
+class Storage
+{
```

Comment:
please keep code style according to JS styleguide, where is `{` should be on same line with classe, var or function definition